CHANGELOG.md

```diff
@@ -2,6 +2,35 @@
 
 Full documentation for rocDecode is available at [https://rocm.docs.amd.com/projects/rocDecode/en/latest/](https://rocm.docs.amd.com/projects/rocDecode/en/latest/)
 
+## rocDecode 0.9.0 (unreleased)
+
+### Changed
+
+* AMD Clang++ is now the default CXX compiler.
+* `rocDecode-setup.py` setup script updates to common package install
+
+### Removed
+
+* 
+
+### Resolved issues
+
+* 
+
+### Tested configurations
```

Comment:
Tested configurations need to be removed from the changelogs. If the information is important for the end user, then let me know and I'll put it in the appropriate place in the documentation.

Reply:
This would be a good to have detail for end-user

Reply:
OK. It'll be in the doc. We can remove it from here later.